pyproject.toml

```diff
@@ -28,6 +28,7 @@ classifiers = [
 requires-python = ">=3.8"
 dependencies = [
     "xarray >= 2022.11.0",
+    "pyproj >= 2.6.1.post1",
```

Comment:
Any specific reason for this version? NEP29?

Reply:
No specific reason, I copied that from GeoPandas's `pyprojet.toml` (main branch). Do you know what would be the minimal supported version here?

Reply:
If we stick to NEP29 geopandas also uses, it would be 3.0.0 (I need to update geopandas prior to the next release as well.